src/main/java/org/web3j/crypto/WalletFile.java

```diff
@@ -62,6 +63,43 @@ public void setVersion(int version) {
         this.version = version;
     }
 
+    @Override
+    public int hashCode() {
```

Comment:
With the hashCode and equals methods, please can you use the same implementations as per https://github.com/web3j/web3j/blob/master/src/main/java/org/web3j/protocol/core/methods/response/TransactionReceipt.java#L159